tests/components/zwave_js/conftest.py

```diff
@@ -608,6 +610,8 @@ async def disconnect():
         client.listen = AsyncMock(side_effect=listen)
         client.disconnect = AsyncMock(side_effect=disconnect)
         client.driver = Driver(client, controller_state, log_config_state)
+        node = Node(client, copy.deepcopy(controller_node_state))
+        client.driver.controller.nodes[node.node_id] = node
```

Comment:
Why not use the `controller_node` fixture in the tests that need this?

Reply:
we could do that but I thought this would be a more accurate representation of a network

Reply:
Should we remove that fixture then?